### DISCUSSION TITLE
[Design] New Hangout Flow - Scheduling a new Hangout

### DISCUSSION THREAD

**Discussion by lpatmo** [codebuddies/frontend#139] [Ideas]
Design: TODO

**Comment by lpatmo** [comment]
[Inspiration] I know we've already discussed some of our own fields, but perhaps we can also derive some inspiration from this scheduler I just noticed on Twitch:
![image](https://user-images.githubusercontent.com/4512699/90992605-bc30e500-e565-11ea-8276-5479cbbe1e95.png)

**Reply by lpatmo** [reply]
... maybe this is more useful as inspiration for a "general availability" field, but I like the ability to check off different days of the week.

**Reply by BethanyG** [reply]
I like this a lot!  Maybe instead of category, CodeBuddies would have that "I am looking to _____ on _____" that we've been discussing on the other thread.
